### PR TITLE
fix: return no content for no avatar

### PR DESCRIPTION
**Pull request by pyphilia**
Hopefully it will solve this problem
```
GET /members/:id/avatar/:size
'S3_FILE_NOT_FOUND' captured as exception with message 'S3_FILE_NOT_FOUND'
environment: production  transaction: GET /members/:id/avatar/:size
```

- [x] test on dev
- [x] write tests

### CODE REVIEW

File: src/services/member/plugins/thumbnail/schemas.ts [hunk 1]
```diff
@@ -13,21 +13,22 @@ export const upload = {
...
+  params: customType.StrictObject({
+    id: customType.UUID(),
+    size: Type.Enum(ThumbnailSize, { default: ThumbnailSize.Medium }),
+  }),
+  querystring: customType.StrictObject({
+    replyUrl: Type.Boolean({ default: false }),
+  }),
   response: {
-    [StatusCodes.OK]: errorSchemaRef,
+    [StatusCodes.OK]: {},
```
Comment (pyphilia) [inline src/services/member/plugins/thumbnail/schemas.ts:30]: Check if this is okay.

File: src/services/member/plugins/thumbnail/schemas.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const download = {
     replyUrl: Type.Boolean({ default: false }),
   }),
   response: {
-    [StatusCodes.OK]: {},
+    [StatusCodes.OK]: Type.String({ description: 'Url string of the avatar if replyUrl is true' }),
```
Comment (spaenleh) [inline src/services/member/plugins/thumbnail/schemas.ts:30]: Not to be changed now, but do we really need to support replying with the file as a blob ? i.e. can we remove the `replyUrl` param and always rely with the url instead ?

File: src/services/member/plugins/thumbnail/test/index.test.ts [hunk 3]
```diff
@@ -50,87 +56,133 @@ jest.mock('@aws-sdk/lib-storage', () => {
...
+
+    it('Get member avatar', async () => {
+      const member = await saveMember(MemberFactory({ extra: { hasAvatar: true } }));
 
       const response = await app.inject({
         method: HttpMethod.Get,
         url: `members/${member.id}/avatar/${ThumbnailSize.Small}`,
       });
-
+      console.log(response);
```
Comment (spaenleh) [inline src/services/member/plugins/thumbnail/test/index.test.ts:78]: remove ?

File: src/services/member/plugins/thumbnail/schemas.ts (hunk 2 above)
Comment (pyphilia) [reply src/services/member/plugins/thumbnail/schemas.ts:30]: Yes, I think we never use one of the case.

**Pull request merged**